src/main/webapp/util/UrlConstants.js

```diff
@@ -0,0 +1,71 @@
+const context = process.env.NODE_ENV !== 'test' ? component.contextPath : 'http://localhost';
```

Comment:
nit: I generally prefer non-negative logic blocks when possible - it's just as easy and more legible to invert this:
```suggestion
const context = process.env.NODE_ENV === 'test' ? 'http://localhost' : component.contextPath;
```